.env.example

```
@@ -24,6 +24,11 @@ POSTGRES_PASSWORD=shelfy
 # MinIO local bootstrap credentials
 MINIO_ROOT_USER=minioadmin
 MINIO_ROOT_PASSWORD=minioadmin
+MINIO_ACCESS_KEY=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ACCESS_KEY key should go before the MINIO_ROOT_PASSWORD key

(UnorderedKey)